cmd/oras/root/push.go

```
@@ -185,7 +185,7 @@ func runPush(cmd *cobra.Command, opts *pushOptions) error {
 	if err != nil {
 		return err
 	}
-	defer store.Close()
+	defer func() { _ = store.Close() }()
```

We need to return the error if `Close()` fails